js/index.js

```diff
@@ -1,7 +1,106 @@
 // Iteration 1: Names and Input
+let hacker1 = "Sabela";
+console.log(`The driver's name is ${hacker1}`)
 
+let hacker2 = "Nina";
+console.log(`The navigator's name is ${hacker2}`)
 
 // Iteration 2: Conditionals
+if (hacker1.length > hacker2.length) {
+    console.log(`The driver has the longest name, it has ${hacker1.length} characters.`)
+  } else if (hacker1.length < hacker2.length) {
+    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`)
+  } else {
+    console.log(`Wow, you both have equally long names, ${hacker1.length} characters!`)
+  }
 
+// Iteration 3: Loops
+/* 3.1 Print all the characters of the driver's name, separated by a space and [in capitals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toUpperCase)
+  i.e. `"J O H N"` */
 
-// Iteration 3: Loops
\ No newline at end of file
+hacker1 = hacker1.toUpperCase().split("").join(" "); 
```

Comment:
Please be consistent in the use of semicolons. Either use them at the end of each statement or don't.

Reply:
Thanks, I unified.  
Still not sure when semicolons are mandatory or not.
Is is "best practices"to use them or not?